src/V3AstNodes.h

```diff
@@ -7107,6 +7107,94 @@ class AstPowUS : public AstNodeBiop {
     virtual int instrCount() const { return widthInstrs() * instrCountMul() * 10; }
     virtual bool signedFlavor() const { return true; }
 };
+class AstPreAdd : public AstNodeBiop {
+public:
+    AstPreAdd(FileLine* fl, AstNode* lhsp, AstNode* rhsp)
+        : ASTGEN_SUPER(fl, lhsp, rhsp) {
+            dtypeFrom(lhsp);
+        }
+    ASTNODE_NODE_FUNCS(PreAdd)
+    virtual AstNode* cloneType(AstNode* lhsp, AstNode* rhsp) {
+        return new AstPreAdd(this->fileline(), lhsp, rhsp);
+    }
+    virtual void numberOperate(V3Number& out, const V3Number& lhs, const V3Number& rhs) {
+        out.opAdd(lhs, rhs);
```

Comment:
A standard operate would get wrong value so can't use this standard function - sorry missed this earlier.
```suggestion
        V3ERROR_NA; // Need to modify lhs
```